share/jupyter/kernels/xq/kernel.json

```diff
@@ -1,10 +0,0 @@
-{
-  "display_name": "Qt 5.12 (xq)",
-  "argv": [
-      "xeus-qt",
-      "-f",
-      "{connection_file}"
-  ],
-  "language": "c++",
-  "metadata": { "debugger": false }
-}
```

Comment:
Do we need this, shouldn't this be done using `configure_file` like all other kernels ?